src/main/java/org/spongepowered/common/world/SpongeProxyBlockAccess.java

```diff
@@ -72,8 +72,9 @@ public TileEntity getTileEntity(BlockPos pos) {
 
     @Override
     public IBlockState getBlockState(BlockPos pos) {
-        if (this.processed.containsKey(pos)) { // first just check if there's already a pos list built.
-            return this.processed.get(pos);
+        IBlockState value = this.processed.get(pos);
```

Comment:
Fluff that isn't necessary. We're not actually using this class since it's been rendered unnecessary by the PhaseTracker system. It used to be required but was eliminated from use since the bugs that were introduced are now gone.

Reply:
@gabizou can he just remove it then?

Reply:
Yep, can be removed.

Reply:
There's also a Mixin of this class (thus an entry in mixins.forge.core.json). I think I will leave this for the OCD issue.